package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "definitely-typed",
-    "version": "0.0.3",
+    "version": "1.0.0",
```

Comment:
This seems like a mistake

Reply:
@connor4312 

This PR changes a function to a class. So this is breaking change and should the major version be updated?
https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package

If this can be minor bug fix, I will fix the version to `0.0.4` !

Thank you.

Reply:
No the version of definitely-typed itself should not change just because tar updated. Maybe you wanted to update the version of the tar package typings only